msfits/MSFits/MSFitsOutput.cc

```diff
@@ -678,8 +698,15 @@ std::shared_ptr<FitsOutput> MSFitsOutput::_writeMain(Int& refPixelFreq, Double&
     // DATE
     desc.addField("date1", TpFloat);
     desc.addField("date2", TpFloat);
-    // BASELINE
-    desc.addField("baseline", TpFloat);
+    // BASELINE if maximum antenna number < 256
+    // SUBARRAY, ANTENNA1 and ANTENNA2 otherwise
```

Comment:
Maybe useful to refer to the relevant section of the UV-fits standard here?

Reply:
Done